src/store/store.ts

```diff
@@ -781,6 +781,8 @@ export const useStore = defineStore('store', {
 
     /** Whether the subject correction filing has any sections in editing mode. */
     isCorrectionEditing (): boolean {
+      if (!this.isCorrectionFiling) return false
```

Comment:
Because special resolutions use some of the flags below, this getter was True for special resolutions. Not only does this make the getter name wrong, but in App.vue, this was called:

![image](https://github.com/user-attachments/assets/b60a433b-1ef8-47d7-b58f-d38e1bfece30)